pages/index.tsx

```diff
@@ -1,35 +1,152 @@
 import Link from "next/link";
```

Comment:
I suggest that you move the code related to the workbench to a separate file and just reuse the component here.

Reply:
alright, thanks!